### ISSUE TITLE
Adding to cart fails

### ISSUE THREAD

**Issue by wlrd** [zweed4u/new_supr3m31059#8]
I am seeing the following error when trying to add to the card - Response Empty! - Problem Adding to Cart! [[Supreme®/Hanes® Tagless Tees (3 Pack)]] FAILED!.

Seems like it keeps hitting this line because the request is failing. 
https://github.com/zweed4u/Supreme/blob/dc785e55b9f596a5e44f4450a1886173984478df/supreme.py#L157

**Comment by zweed4u** [comment]
@wlrd 
So yeah, that's what it's hitting but why?
I'm able to add every product to cart barring 'XLarge' size (which I assume you meant to mention).
Looking into it further it's rectified by adjusting the conditional for matching.
Just pushed updates: [19b04f7](https://github.com/zweed4u/Supreme/commit/19b04f7b2222cf2ab9e154c70616b685081156d3) and [324b570](https://github.com/zweed4u/Supreme/commit/324b570b57f27c07af1aee00d0a51829b58e3718); should work. 

Tested full size run and both colorways of tee
Let me know if it works for you.

**Comment by wlrd** [comment]
Seems to work now! Thanks.

**Comment by zweed4u** [comment]
Awesome!

**Comment by ferrucc-io** [comment]
Hitting the same problem here!
Here's my console output with a dummy product:

`[Thread-1]] Pin Up Chino :: 34 :: Khaki :: 1 :: Thread initialized!`

`[[Thread-1]] 2018-05-20 23:31:22.204926 EST :: [[Pin Up Chino Pant]] 303214 found ( MATCHING ITEM DETECTED )`

`[[Thread-1]] 2018-05-20 23:31:22.205214 EST :: Selecting [[Pin Up Chino Pant]] ( 303214 )`

`[[Thread-1]] 2018-05-20 23:31:22.366081 EST :: Selecting size for [[Pin Up Chino Pant]] - 34 ( Khaki )  ( 44991 )`

`[[Thread-1]] 2018-05-20 23:31:22.366789 EST :: Adding [[Pin Up Chino Pant]] to cart...`

`[[Thread-1]] 2018-05-20 23:31:22.548481 EST :: Response Empty! - Problem Adding to Cart! [[Pin Up Chino Pant]] FAILED!`

The only thing I changed in the code, to make it work for the EU one is that I deleted the "State" variable as it's not there if you're from outside the US. But as the script doesn't add the product to the cart it shouldn't be hitting errors related to this changes yet.

I tried to check the page source code of the website and the POST request still looks like the one in the  scripts.

I also tried to run the python 2 version, but got exactly the same output.
